presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedSystemAccessControl.java

```diff
@@ -138,13 +140,20 @@ public SystemAccessControl create(Map<String, String> config)
         {
             requireNonNull(config, "config is null");
 
-            String configFileName = config.get(SECURITY_CONFIG_FILE);
+            Bootstrap bootstrap = new Bootstrap(
+                    new FileBasedAccessControlModule("fileBasedSystemAccessControlFactory"));
```

Comment:
This module is for connector level access control. Please do not use it here. Instead please:
```
new Bootstrap(() -> configBinder(binder).bindConfig(FileBasedAccessControlConfig.class));
```

Reply:
Hey @kokosing , I have updated the pull request with suggested changes and about the CLA I already sent a mail attaching CLA pdf document filling the details and signing it.